Source/Scene/Model/ModelDrawCommand.js

```diff
@@ -56,25 +56,22 @@ function ModelDrawCommand(options) {
   const isTranslucent = command.pass === Pass.TRANSLUCENT;
   const isDoubleSided = runtimePrimitive.primitive.material.doubleSided;
   const usesBackFaceCulling = !isDoubleSided && !isTranslucent;
+  const hasSilhouette = renderResources.hasSilhouette;
 
-  const hasOpaqueAndTranslucentFeatures =
-    renderResources.styleCommandsNeeded ===
-    StyleCommandsNeeded.OPAQUE_AND_TRANSLUCENT;
-
-  // CPUStylingPipelineStage sets the pass to translucent if the style commands
-  // needed are all translucent, so if the command was already translucent,
-  // there's no need to derive a new command.
+  // If the command was already translucent, there's no need to derive a new
+  // translucent command. As of now, a style can't change an originally
+  // translucent feature to opaque since the style's alpha is modulated,
+  // not replaced. When this changes, we need to derive new opaque commands
+  // in initialize().
   //
-  // As of now, a style can't change an originally translucent feature to opaque
-  // since the style's alpha is modulated, not a replacement. When this changes,
-  // we need to derive new opaque commands in initialize().
-  const needsTranslucentCommand =
-    hasOpaqueAndTranslucentFeatures && !isTranslucent;
+  // Silhouettes for primitives with both opaque and translucent features
+  // are not yet supported.
```

Comment:
Note: this was implicit in the design decisions in #10697. We chose not to derive translucent silhouette commands because it was going to get complicated, and we figured that the case wouldn't happen frequently enough to be worth the complexity. We can always add support later, though we would have to redesign the derived commands system to handle it.